problemsets-mat223/modules/module7-exercises.tex

```diff
@@ -47,6 +167,45 @@
 			\mat{2\\-8\\2} \cdot \left(\mat{x\\y\\z}-\mat{1\\7\\0}\right).
 		\]
 		Find $\mathcal P\cap\mathcal Q\cap\mathcal R$.
-
+		\begin{solution}
+			We first express all the planes in the same form (any form works but we choose to use the equation form).\\
+			$\mathcal{P}$ is already expressed in this form: $2x+4y-4z=7$.\\
+			We are given $\mathcal{Q}$ in the form \[\vec x=t\mat{-1\\2\\0}+s\mat{5\\0\\2}+\mat{0\\7\\1}\]
+			which yields 
+			\begin{align*}
+				x&=-t+5s\\
```

Comment:
Obviously, these newlines should not be removed, since they are part of the environment ;-).